content/es/api_gateway.md

```diff
@@ -0,0 +1,16 @@
+---
+title: API Gateway
+status: Completed
+category: technology
+---
+
+## ¿Qué es esto?
+Un [API Gateway](/application_programming_interface/), API de pasarela, es una herramienta que concentra diferentes aplicaciones APIs, centralizando a todas en un solo lugar. Esto permite a las organizaciones mover funciones clave, como ser la autenticación y la autorización o limitando el número de peticiones entre aplicaciones, a una ubicación de administración centralizada. Un API Gateway funciona como una interface común para los que la consumen (usualmente aplicaciones externas).
```

Comment:
"Pasarela API", sigue la referencia https://www.connecting-software.com/es/blog/what-is-an-api-gateway-how-it-can-actually-deliver-practical-results/